content/tutorial/tutorial.md

```diff
@@ -164,33 +162,33 @@ return React.createElement('div', {className: 'shopping-list'},
 );
 ```
 
-[See full expanded version.](babel://tutorial-expanded-version)
+[Lihat versi expanded lengkap.](babel://tutorial-expanded-version)
```

Comment:
`Lihat versi lengkap` is enough for this I think, what do you think?

Reply:
Yeah, this is a little confusing. There's no word that could represent **expanded** in Bahasa. I am afraid to delete it because **expanded** in this context (React component) means more than "lengkap".